packages/formation-react/scripts/build.js

```diff
@@ -19,9 +19,9 @@ function flattenRequires(bufferString) {
     recast.print(
       recast.visit(recast.parse(bufferString), {
         visitCallExpression: function(path) {
+          this.traverse(path);
```

Comment:
The way this apparently works is to visit all top level call expressions, then `traverse` will traverse deeper into the AST and find more more call expressions. This is fine when all the require statements are at the first level of the AST, but after the recent Babel update, some of them are nested in an interop function. This change makes it so that all call expressions are traversed.